oasislmf/cli/exposure.py

```diff
@@ -217,6 +223,8 @@ def action(self, args):
                 loss_factor, total_gul, total_il, total_ri_ceded)
 
         print_dataframe(all_losses_df, frame_header=header, string_cols=all_losses_df.columns)
+        if output_file:
+            all_losses_df.to_csv(output_file, index=False)
```

Comment:
May be not necessarily here if the table only contains numbers and no text, but there is no harm in adding argument to set encoding type

    all_losses_df.to_csv(output_file, index=False, encoding='utf-8')

Reply:
Fixed. Note that utf-8 is the default encoding in Python 3 anyway.